### DISCUSSION TITLE
更换开机自启快捷方式所在位置

### DISCUSSION THREAD

**Discussion by CloudyEagle** [PBH-BTN/PeerBanHelper#570] [讨论]
我有手动管理开启自启的习惯发现安装后开机自启的快捷方式在用户自己的文件夹中，大致路径是
`C:\Users\%UserName%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup`

不过我会把所有开机自启的快捷方式放到另一个开机自启文件夹中，路径是
`C:\ProgramData\Microsoft\Windows\Start Menu\Programs\StartUp`

请问安装时能否就直接装到后一个文件夹中，还是安装包的提权不够，放不进去，需要用户手动修改？

**Comment by Gaojianli** [comment]
这个的区别是：“为当前用户安装”和”为所有用户安装“

**Reply by CloudyEagle** [reply]
我安装软件的时候都是选”为所有用户安装“的，但生成的开机自启图标还是在个人用户的文件夹里的样子。

**Comment by Ghost-chu** [comment]
PBH 安装程序默认总是安装为用户应用，只有安装为系统服务才需要管理员权限。

除非你在多个用户那里都需要 PBH，否则总是建议安装到用户目录，这样不需要管理员权限。

如果是很多个用户共用，请考虑安装为系统服务。

**Reply by CloudyEagle** [reply]
我之前安装成系统服务过，但是升级后之前生成的token好像用不了，所以就用这种拙劣的方式了。

**Reply by Ghost-chu** [reply]
升级的时候也要勾选安装为系统服务。

installer 没有刻意优化过，仅仅是能用的水平。

**Reply by CloudyEagle** [reply]
明白了，那我关闭这个讨论了。
